backend/app/domain/services/base/model.py

```diff
@@ -553,8 +555,6 @@ async def conversation_with_buffer_memory(
         prompt: str,
         num_answers: int,
     ):
-        print("Selected model was", model_name)
-        print("Selected model was", model_name[provider]["model_name"])
```

Comment:
Rafa do we really want to delete these prints, I left them for logging purposes, but If you feel we don't need those .. then let's delete them.

Reply:
I think those have been there for quite a while to be honest. I do think they give some info, but we should be logging them instead, hence why I deleted them/